### PR TITLE
SCSS importer did not support enough import syntaxes

### PR DESCRIPTION
**Pull request by plumpNation**
For instance node_modules

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
+    "version": "8.4.0",
     "author": "Gavin King <gavin.king@itslearning.com>",
     "license": "MIT",
     "scripts": {
         "build": "webpack-cli",
         "dev": "webpack-cli --watch",
         "test": "yarn build && karma start"
     },
+    "browserslist": [
+        "last 4 versions"
```
Comment (furstenberg) [inline package.json]: If I'm not mistaken this will give us IE8 and newer support.

http://browserl.ist/?q=last+4+versions

Comment (EsmeraldiBejolli) [reply package.json]: I am afraid that this is going to add a loot extra css code that we dot not need

Comment (furstenberg) [reply package.json]: Perhaps http://browserl.ist/?q=last+2+versions%2C+not+dead ?

Comment (EsmeraldiBejolli) [reply package.json]: ie 11 is supported by default I guess
http://browserl.ist/?q=defaults

**Pull request merged**